lib/fat_free_crm/core_ext/alias_method_chain.rb

```diff
@@ -0,0 +1,24 @@
+class Module
+
+  def alias_method_chain(target, feature)
```

Comment:
Potentially not needed anymore with the swap to prepend

Reply:
@CloCkWeRX this is needed due to gem dependencies which are not yet using the prepend syntax. Build will fail without it. I have converted all FFCRM code to use prepend.

Reply:
Lets ticket for removal in 0.16.0 then, and maybe add some deprecation warnings like http://api.rubyonrails.org/v5.0/classes/ActiveSupport/Deprecation/Reporting.html lets you do

Reply:
@CloCkWeRX removal will not be possible unless we remove all alias_method_chain usage from all third-party gem dependencies, which we do not control. Unfortunately this patch will be here for quite awhile.

Reply:
Semver will save us, even if it means that we cut off a bunch of older gems that aren't 0.16.0+ compatible.

Reply:
@CloCkWeRX we should raise a separate issue to do that and not make it a blocker for the 0.16.0 release. We cannot guarantee the timing at which other gem owners will release a version that removes `alias_method_chain`